src/components/Vault/VaultData.tsx

```diff
@@ -43,22 +45,33 @@ export const VaultTvl: FC<VaultProps> = (props) => {
 
   return (
     <Skeleton isLoading={isLoadingId || isLoading}>
-      <Currency symbol="$">{tvl ?? 0}</Currency>
+      <Currency symbol="$" abbreviate>
```

Comment:
abbreviate is not defined. Is this what you desired?

Reply:
You can define boolean props like this -- it's the same as writing `abbreviate={true}`